File: PersonnelManagement/PersonnelManagement/Front/classes/decree/decree.ts
export default class Decree {
    constructor() {
        this.id = 0;
        this.name = "";
        this.signed = false;
        this.declined = false;
        this.dateactive = new Date();
        this.datesigned = new Date();
        this.user = 0;
        this.nickname = "";
        this.number = "";
        this.historycal = 0;
    }
    id: number;
    name: string;
    signed: boolean;
    declined: boolean;
    dateactive: Date;
    datesigned: Date;
    user: number;
    nickname: string;
    number: string;
    historycal: number;
}

File: PersonnelManagement/PersonnelManagement/Models/Decree.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Decree
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public sbyte Signed { get; set; }
        public sbyte Declined { get; set; }
        public DateTime? Dateactive { get; set; }
        public DateTime? Datesigned { get; set; }
        public int? User { get; set; }
        public string Nickname { get; set; }
        public string Number { get; set; }
        public int Historycal { get; set; }
    }
}
